client/rest-high-level/src/main/java/org/elasticsearch/client/security/KibanaEnrollmentResponse.java

```diff
@@ -18,21 +18,21 @@
 
 public final class KibanaEnrollmentResponse {
 
-    private SecureString password;
+    private SecureString token;
```

Comment:
@azasypkin would Kibana want to know the token name in addition to the value ? ( https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-create-service-token.html )

Reply:
A explicit token name might be convenient for kibana, but the token value itself also has the token name embedded. The token value takes the format of `\0\1\0\1elastic/kibana-system/$tokenName:$tokenSecret` and is base64 encoded.

Reply:
> @azasypkin would Kibana want to know the token name in addition to the value ? ( https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-create-service-token.html )

Yeah, I think it'd be better to have one (to log it least).

Reply:
From what I can gather we only need the service account token to authenticate so don't think we need the name at all.

Reply:
Right, we don't need it in the code, but I thought it'd be beneficial to log it anyway, to know which Kibana triggered generation of which token, for troubleshooting purposes.

Reply:
To be honest, for consistency, it might make sense to mirror the create service token API anyways:

```
{
  "token": {
    "name": "token1",
    "value": "AAEAAWVsYXN0aWM...vZmxlZXQtc2VydmVyL3Rva2VuMTo3TFdaSDZ" 
  }
}
```

Learn once, use anywhere, etc.

Reply:
coo, I'll make the change to return the name too, thanks for weighing in folks